README.md

```diff
@@ -38,251 +11,123 @@ After you're finished please remove all the comments and instructions!
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
+not ready for deployment
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
+## 👥 Authors <a name="authors">Iheagwara Augustine</a>
 
-👤 **Author1**
+.
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Iheagwara Augustine**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@githubhandle](https://github.com/stino-x)
+- Twitter: [@twitterhandle](https://twitter.com/Stino_x_Uncaged)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
 
-## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🔭 Future Features <a name="future-features"></a>
 
-<!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+- [ ] **[carousel]**
+- [ ] **[new sections]**
 
-I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
+## 🤝 Contributing <a name="contributing"></a>
+none to note
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
 
-  - [Answer_1]
+- **[what did you use for positioning]**
 
-- **[Question_2]**
+  - [flexbox]
 
-  - [Answer_2]
+- **[how did you get the button to change color?]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+  - [hover]
 
-<!-- LICENSE -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ]  The link to your MIT LICENSE is broken. Kindly add a valid LICENSE. The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. It would be nice to fix it. 👍 

You can follow the instructions on the right side of the article attached below to set up a valid _**LICENSE**_ 

https://choosealicense.com/licenses/mit/

Reply:
there is already a MIT license file in the project , how do i get a link to a file in the same project? @Gambit142

Reply:
First of all, your MIT file is not completed with information regarding your project and yourself.
Screenshot from your MIT
![image](https://user-images.githubusercontent.com/83596135/212430854-f70a45cc-c3a2-4f4a-9c19-56a914e4174a.png)

Second, you need to link the MIT link from your README with your MIT file.
You should have it like this: [MIT](./MIT.md)